src/app/services/actions/profile-action-creators.ts

```diff
@@ -1,7 +1,14 @@
 import { IQuery } from '../../../types/query-runner';
-import { PROFILE_IMAGE_REQUEST_SUCCESS, PROFILE_REQUEST_ERROR, PROFILE_REQUEST_SUCCESS } from '../redux-constants';
+import { PROFILE_IMAGE_REQUEST_SUCCESS, PROFILE_REQUEST_ERROR, PROFILE_REQUEST_SUCCESS, PROFILE_TYPE_SUCCESS } from '../redux-constants';
 import { authenticatedRequest, isImageResponse, parseResponse } from './query-action-creator-util';
 
+export function profileTypeSuccess(response: any): any {
+  return {
+    type: PROFILE_TYPE_SUCCESS,
+    response,
+  }
+}
+
```

Comment:
not tested because it is being called within a function that is already being tested.